gandalf/src/test/resources/allPresent.json

```diff
@@ -0,0 +1,16 @@
+{
+  "android": {
+    "alert": {
+      "message": "Alert Message",
+      "blocking": true
+    },
+    "optionalUpdate": {
+      "optionalVersion": "6",
+      "message": "A new version of the application is available, please click below to update to the latest version."
+    },
+    "requiredUpdate": {
+      "minimumVersion": "7",
+      "message": "A new version of the application is available and is required to continue, please click below to update to the latest version."
+    }
+  }
+}
```

Comment:
Is this used?

Reply:
Nope, I was planning on it but it didn't seem necessary in the end. I'll get it out of there 👍